ablySpec/RealtimeClientChannel.swift

```diff
@@ -186,6 +187,56 @@ class RealtimeClientChannel: QuickSpec {
             // RTL6
             describe("publish") {
 
+                // RTL6a
+                it("should encode messages in the same way as the RestChannel") {
+                    let data = ["value":1]
+                    var restEncodedMessage: ARTMessage?
+                    var realtimeEncodedMessage: ARTMessage?
+
+                    let rest = ARTRest(options: AblyTests.commonAppSetup())
+                    let restChannel = rest.channels.get("test")
+
+                    let restBlock: @convention(block) (AspectInfo, ARTMessage) -> Void = { info, message in
+                        let invocation = info.originalInvocation()
+                        invocation.invoke()
```

Comment:
Yes, you right, It is doing a copy of the `message`. I will try to use the `NSInvocation.getReturnValue`. Ty.